tests/test_slurm_command_gen_strategy.py

```diff
@@ -56,6 +57,29 @@ def strategy_fixture(slurm_system: SlurmSystem) -> SlurmCommandGenStrategy:
     return strategy
 
 
+@pytest.fixture
+def test_run_fixture() -> TestRun:
+    test = Mock()
+    test.cmd_args = {
+        "docker_image_url": "fake",
+        "repository_url": "fake",
+        "repository_commit_hash": "fake",
+        "data_impl": "mock",
+        "data_dir": "/fake/data_dir",
+    }
+    test.extra_env_vars = {"TEST_VAR_1": "value1"}
+    test.extra_cmd_args = "extra_args"
+
+    tr = TestRun(
+        test=test,
+        num_nodes=2,
+        nodes=[],
+        output_path=Path("/tmp/output"),
```

Comment:
Let's use pytest's fixture (`tmp_path`) here to cleanup after test run automatically. Same for `"data_dir"`.